src/helpers/constants.ts

```diff
@@ -1 +1,18 @@
 export const ETH_CONTRACT = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee';
+
+export const AUTHS = {
+  '0x594a81b66c3aa2c64577916f727e1307b60c9d6afa80b6f5ca3e3049c40f643':
+    'Vanilla',
+  '0x6ad07205a4d725c5c2b10c4f5fbdfaaa351c742fce7a5a22b2b56fd8d5afd62':
+    'Ethereum signature'
```

Comment:
```suggestion
  '0x6ad07205a4d725c5c2b10c4f5fbdfaaa351c742fce7a5a22b2b56fd8d5afd62':
    'Vanilla',
  '0x594a81b66c3aa2c64577916f727e1307b60c9d6afa80b6f5ca3e3049c40f643':
    'Ethereum signature'
```

Those are the other way around.